### PR TITLE
Dropdown and icon buttons refactor; partially closes #2725.

### LINKED ISSUE #2725: Refactor styles
Currently, we have 2 approaches when applying border to a button or icon button in hover/focust state:
1. using border
2. using box-shadow

I think we need to make it uniform, such that it will use only `box-shadow`, the reason being that it won't affect the element inner sizing.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Partially closes https://github.com/eventespresso/event-espresso-core/issues/2725

### CODE REVIEW

**Comment by nicolad** [comment]
image

File: assets/src/application/ui/display/icons/svgs/sort.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	aria-hidden="true"
+	focusable="false"
+	xmlns="http://www.w3.org/2000/svg"
+	version="1.1"
+	height="1.25rem"
+	viewBox="0 0 20 20"
+	width="1.25rem"
+>
+<path d="M11 7h-10l5 7zM9 14h10l-5-7z" />
+</svg>
```
Comment (nicolad) [inline assets/src/application/ui/display/icons/svgs/sort.svg:11]: got it from dashicons, hope it's fine:

https://github.com/WordPress/dashicons/blob/master/sources/svg/sort.svg

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
All good in FF:

image

File: assets/src/application/ui/display/icons/svgs/sort.svg (hunk 1 above)
Comment (tn3rb) [reply assets/src/application/ui/display/icons/svgs/sort.svg:11]: love it !!!

File: assets/src/application/ui/input/Button/IconButton/types.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { withLabelProps, withTooltipProps } from '../../../display';
+import { IconButtonProps as IconButtonAdapterProps } from '@infraUI/inputs';
+
+export interface IconButtonProps
+	extends Omit<IconButtonAdapterProps, 'aria-label'>,
+		Partial<withLabelProps>,
+		Partial<withTooltipProps> {
+	'aria-label'?: string;
+	borderless?: boolean;
+	color?: 'white' | 'black';
```
Comment (tn3rb) [inline assets/src/application/ui/input/Button/IconButton/types.ts:10]: hmmmm... good for now but will change later... black is usually too strong of a contrast, but not going to worry about it now

File: assets/src/application/ui/layout/dropdownMenu/DropdownMenu.tsx [hunk 3]
```diff
@@ -11,12 +11,12 @@ const DropdownMenu: React.FC<DropdownMenuProps> = ({ children, className, menuLi
 	return (
 		<Menu>
 			{({ isOpen, onClose }) => (
-				<>
+				<div className='ee-dropdown-menu'>
```
Comment (tn3rb) [inline assets/src/application/ui/layout/dropdownMenu/DropdownMenu.tsx:14]: 👍

File: assets/src/application/ui/input/Button/IconButton/types.ts (hunk 2 above)
Comment (nicolad) [reply assets/src/application/ui/input/Button/IconButton/types.ts:10]: yes, that was just to have smth as an example, will adjust in the future as per our needs

**Pull request merged**